cmd/milmove/serve.go

```diff
@@ -129,7 +129,10 @@ func initServeFlags(flag *pflag.FlagSet) {
 	// Verbose
 	cli.InitVerboseFlags(flag)
 
-	// Sort command line flags
+	// Feature Flags
+	cli.InitFeatureFlags(flag)
+
+	// Sort flags
```

Comment:
I think the original comment here was better.
```suggestion
	// Sort command line flags
```